mscore/pianolevelschooser.cpp

```diff
@@ -11,23 +12,28 @@ namespace Ms {
 PianoLevelsChooser::PianoLevelsChooser(QWidget *parent)
       : QWidget(parent)
 {
-      _levelsIndex = 0;
+      setupUi(this);
 
-      QGridLayout* layout = new QGridLayout;
+      _levelsIndex = 0;
 
-      levelsCombo = new QComboBox;
       for (int i = 0; PianoLevelsFilter::FILTER_LIST[i]; ++i) {
             QString name = PianoLevelsFilter::FILTER_LIST[i]->name();
             levelsCombo->addItem(name, i);
             }
 
-      layout->addWidget(levelsCombo, 0, 0, 1, 1);
-
-      setLayout(layout);
-
       connect(levelsCombo, SIGNAL(activated(int)), SLOT(setLevelsIndex(int)));
+      connect(setEventsBn, SIGNAL(clicked(bool)), SLOT(setEventDataPressed()));
 }
 
+//---------------------------------------------------------
+//   setStaff
+//---------------------------------------------------------
+
+void PianoLevelsChooser::setStaff(Staff* staff)
+      {
+      _staff = staff;
```

Comment:
This can be defined in .h file along with `staff()`.

Reply:
Wouldn't it be better to have it in the .cpp file if it does not need to be seen outside the module?